app/src/main/java/com/google/droidjump/GameFailureFragment.java

```diff
@@ -51,6 +57,10 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
                 NavigationHelper.navigateToFragment(activity, new HowToPlayFragment()));
         ((LinearLayout) rootView.findViewById(R.id.droid_draw_view)).addView(new DroidStartView(activity));
         NavigationHelper.addOnBackPressedEventListener(activity, new StartFragment());
+        TextView textView = rootView.findViewById(R.id.score_text_view);
+        String scoreInfo = "score: " + ((Integer) currentLevelScore).toString() +
+                "\nyour best score: " + ((Integer) currentLevelMaxScore).toString();
```

Comment:
I would prefer dong a string.format here. Also extract "score" and "best score " into variables.

I am not sure if you need to cast currentLevelScore to Integer and then toString()

Reply:
Done